base/doc/ltnews41.tex

```diff
@@ -149,6 +149,26 @@ \section{New or improved commands}
 
 \section{Code improvements}
 
+\subsection{Refinement of \cs{MakeTitlecase}}
+
+We introduced \cs{MakeTitlecase} as a late addition to the June
+2022 release, making use of the improved case code in
+\pkg{expl3}. Unlike upper and lowercasing, making text
+titlecased is more tricky to get right: for example, do you just
+change the first character and leave the rest alone.
```

Comment:
or what? the way written doesn't seem like an example

Reply:
I've reworded